workshop/workshop-information.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Workshop Information
+nav_title: Information
+---
+
+{% if site.workshop_type == "remote" %}
+    {% include_relative workshop-information-files/workshop-structure.md %}
+{% elsif site.workshop_type == "in-person" %}
+    {% include_relative workshop-information-files/participant-information.md %}
+{% endif %}
```

Comment:
I feel like most of the content in the "workshop structure" document are relevant to either kind of workshop. We should include that content (minus the "virtual workshop sentence") either way, I think?

Reply:
I've been thinking about the right way to go about this, specifically for TOC reasons. One drawback of using a single file here is that, while it appeases the navbar, you can't dynamically build a TOC based on which contents are actually included in the file. Especially for in-person workshops - if _both_ workshop structure and logistics are included, that's a lot of content and a TOC detailing all the headers would be helpful! 

So, now I'm thinking, what if we move on from this approach entirely, and instead:
- Keep `workshop-structure.md` always in the navbar
- From `HOME.md` (would be done in #123), **link to** the participant logistics page; it wouldn't be in the navbar in this case, but it would loudly linked!
  - Can also add a link in `workshop-structure.md`, like "for more logistical info, see here" or so.  

What do you think?

Reply:
@jashapiro just giving this comment a boost; some of these decisions impact how I tackle next steps, so curious to hear your thoughts!

Reply:
A couple thoughts, I guess. One is manual TOC, which could be fine, as we could add the appropriate sections with `if` statements. You could also consolidate to a single document with all of the logistics in an if statement, no includes.

Another potential thought is to let `jekyll` (or really `kramdown`) generate the TOC, which I'm pretty sure will happen _after_ the includes, but this would require local testing. 

To insert a table of contents, you first make a list to be replaced, then the `{:toc}` directive. So something like:

```
**Table of contents**

* TOC goes here
{:toc}

```

Maybe try that out?

Reply:
Took me a while to find the docs, but here they are: https://kramdown.gettalong.org/converter/html.html#toc

Reply:
Ugh (a good ugh), it just _works_ and it is _beautiful_

Reply:
Should we _always_ do this? Yeah, probably.